src/specfit.jl

```diff
@@ -209,24 +209,30 @@ Get the FWHM of a peak from the fit parameters.
     * `fwhm`: the FWHM of the peak
 """
 function estimate_fwhm(v::NamedTuple)
-    # get FWHM
+    
     f_sigWithTail = Base.Fix2(get_th228_fit_functions().gamma_sigWithTail,v)
     try
-        if v.skew_fraction <= 0.5
-            half_max_sig = maximum(f_sigWithTail.(v.μ - v.σ:0.001:v.μ + v.σ))/2
```

Comment:
In this line, an `Array` is created, from which then a `maximum` is computed. This allocates memory. I replaced this by a `for`-loop that stops when the maximum is found.